tests/application/cms/views/test_data_source_search.py

```diff
@@ -0,0 +1,49 @@
+from bs4 import BeautifulSoup
+
+from tests.models import DataSourceFactory, MeasureVersionFactory
+
+from tests.utils import find_input_for_label_with_text
+
+
+class TestAddDataSourceView:
```

Comment:
Minor niggle - I think 'Search' might be better than 'Add' here.

Reply:
Good catch - copy and paste error from the other tests... Surprised it allowed me to declare the same class twice...